part-1-ajax/test.js

```diff
@@ -1,76 +1,58 @@
-function getCountryName(code) {
-    $.ajax({
-      //uses the three letter code to get the country's full name
-      }
-    })
-  }
 
-  function parseData(country) {
-    //takes the country data
-    //extracts the name, capital, population, and neighboring countries
-    //iterates through the neighbors array to send off second AJAX request
-  }
+$('form').submit(function(event) {
+  event.preventDefault()
+  var input = $(this).serializeArray()[0].value
+  input = input.split(" ").join("+")
+  input = input + "?fullText=true"
+  $('.first').val('');
+
+  $.ajax({
```

Comment:
you want to get in the habit of writing shorter functions - right now you have a function that goes for about 50 lines of code